app/assets/javascripts/media_player_wrapper/mejs4_plugin_playlist_items.es6

```diff
@@ -286,7 +286,12 @@ Object.assign(MediaElementPlayer.prototype, {
      * @return {Object} jQuery object
      */
     getNextItem() {
-      return this.$nowPlayingLi.next('li');
+      let nextItem = this.$nowPlayingLi.next('li');
+      // When next item is not a valid playlist item (e.g. from a deleted item) get the item after that
+      if(nextItem[0].className != 'queue') {
+        nextItem =  nextItem.next('li');
+      }
```

Comment:
Code Climate suggests this change.  What do you think?
```suggestion
      if(nextItem[0].className !== 'queue') {
        nextItem = nextItem.next('li');
      }
```

Reply:
Yeah sounds good :+1: I will add this.